fe1-web/src/features/index.ts

```diff
@@ -17,10 +22,49 @@ export function configureFeatures() {
   const messageRegistry = new MessageRegistry();
   const keyPairRegistry = new KeyPairRegistry();
 
+  const getCurrentLaoUnboound = makeCurrentLao();
+
+  /**
+   * Returns the current lao and throws an error if there is none
+   * @returns The current lao
+   */
+  const getCurrentLao = () => {
+    const currentLao = getCurrentLaoUnboound(getStore().getState());
+
+    if (!currentLao) {
+      throw new Error('Error encountered while accessing storage : no currently opened LAO');
+    }
+
+    return currentLao;
+  };
+
+  /**
+   * Returns the current lao id and throws an error if there is none
+   * @returns The current lao id
+   */
+  const getCurrentLaoId = () => {
+    const currentLaoId = getCurrentLaoIdUnbound(getStore().getState());
+
+    if (!currentLaoId) {
+      throw new Error('Error encountered while accessing storage : no currently opened LAO');
+    }
+
+    return currentLaoId;
+  };
```

Comment:
ideally, these should very much be exported from the `lao` feature, without needing to declare them here. 
Maybe as part of the `laoConfig` below.